### PR TITLE
chore: Add docs for escrowAndAllocateTo

### PR DESCRIPTION
**Pull request by katelynsills**

### CODE REVIEW

File: main/zoe/api/zoe-helpers.md [hunk 1]
```diff
@@ -178,7 +179,7 @@ const { swap } = makeZoeHelpers(zoe);
   return zcf.makeInvitation(hook);
 ```
 
-## zoeHelper.inviteAnOffer(options)
+## zoeHelpers.inviteAnOffer(options)
 - `options` `{offerHook, customProperties, expected}` Optional name 
   properties of the
```
Comment (Chris-Hibbert) [inline main/zoe/api/zoe-helpers.md:184]: This is going away, right? If it's not, we should fix this dangling description

File: main/zoe/api/zoe-helpers.md [hunk 2]
```diff
@@ -225,3 +226,39 @@ const { makeEmptyOffer } = makeZoeHelpers(zoe);
 
 makeEmptyOffer().then(offerHandle => {...})
 ```
+
+## zoeHelpers.escrowAndAllocateTo()
```
Comment (Chris-Hibbert) [inline main/zoe/api/zoe-helpers.md:230]: Please describe the parameter here.

File: main/zoe/api/zoe-helpers.md (hunk 1 above)
Comment (katelynsills) [reply main/zoe/api/zoe-helpers.md:184]: Yeah, I'll fix it, even though it's deprecated

File: main/zoe/api/zoe-helpers.md (hunk 2 above)
Comment (katelynsills) [reply main/zoe/api/zoe-helpers.md:230]: Good point, I forgot

**Pull request merged**